config/config.go

```diff
@@ -443,6 +445,13 @@ func Validate(c *configpb.Configuration) error {
 		return multierror.Append(mErr, MissingFieldError{"Dashboards"})
 	}
 
+	// Each Dashboard must contain at least 1 Tab to do anything
+	for _, dashboard := range c.GetDashboards() {
+		if len(dashboard.DashboardTab) == 0 {
```

Comment:
`dashboard.DashboardTab != ""` is the (faster and) more idiomatic way of writing this.

Reply:
dashboard.DashboardTab is a []*DashboardTab, not a string